package.json

```diff
@@ -12,6 +12,9 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
+        "npm": "2.15.10"
```

Comment:
Oh, and `npm@3` doesn't play well with node `0.10` so we need to merge two other PRs first.